package.json

```diff
@@ -12,6 +12,9 @@
     "test": "mocha --harmony tests/**",
     "test-frontend": "mocha --harmony examples/multiple-fragments-with-custom-amd/test.js",
     "coverage": "node --harmony node_modules/.bin/istanbul cover _mocha -- tests/**",
+    "example:basic": "node --harmony examples/basic",
```

Comment:
can we just remove these commands? because in future we will update more examples and it ll get messy..